clang/www/c_status.html

```diff
@@ -176,7 +176,7 @@ <h2 id="c2y">C2y implementation status</h2>
     <tr>
       <td>Obsolete implicitly octal literals and add delimited escape sequences</td>
       <td><a href="https://www.open-std.org/jtc1/sc22/wg14/www/docs/n3353.htm">N3353</a></td>
-      <td class="none" align="center">No</td>
+      <td class="unreleased" align="center">Clang 21</td>
```

Comment:
https://compiler-explorer.com/z/KxE39czob the "add delimited escape sequences" part is not fully implemented

Reply:
Ah good point, I missed that.